setup.py

```diff
@@ -2,12 +2,15 @@
 
 from distutils.extension import Extension
 from Cython.Distutils import build_ext
+# to get the proper numpy header locations (if using anaconda)
+import numpy as np
 
 ext_modules = []
 ext_modules.append(Extension(
```

Comment:
In general, I'm not sure if this extension should be part of this package at all. It was some extra stuff that I threw in to play around with, but it's not needed for the core function of the package (which is accessing and manipulating the Kitti dataset), and it makes it much more difficult to get things installed. So rather than try to get this Numpy stuff fixed, I think it should just be removed.

Is this extension something you're using, @jasoncorso?